icons/scale-3d.json

```diff
@@ -3,12 +3,14 @@
   "contributors": [
     "lscheibel",
     "ericfennis",
-    "jguddas"
+    "jguddas",
+    "gurtt"
```

Comment:
Normally, we only give contribution for substantial design changes or optimizations, not metadata changes, but @ericfennis and @jguddas may veto me on this one (should they both agree otherwise).

Reply:
Thanks for clarifying 👍 I couldn't find any good documentation for these fields

Reply:
Yess only for design changes

Reply:
I usually add my name when I manually change something inside the `icon.svg`.